src/features/settings/appearance/appearanceSlice.tsx

```diff
@@ -36,7 +37,17 @@ export const OCommentThreadCollapse = {
 } as const;
 
 export type CommentThreadCollapse =
-  (typeof OCommentThreadCollapse)[keyof typeof OCommentThreadCollapse];
+    (typeof OCommentThreadCollapse)[keyof typeof OCommentThreadCollapse];
+
+export const OCommentDefaultSort = {
+  Hot: "Hot",
+  Top: "Top",
```

Comment:
I think it's conventional in this file for these strings (but not the identifiers) to be lowercase. The case is transformed before being displayed to the user.